locales/nn-NO/src/structure.ts

```diff
@@ -340,15 +340,18 @@ export default removeUndefinedLocaleResources({
   /** The text when an unpublish operation failed  */
   'panes.document-operation-results.operation-error_unpublish':
     'Ein feil oppstod under forsøket på å avpublisere dette dokumentet. Dette betyr vanlegvis at det er andre dokument som refererer til det.',
-  /** The text when a generic operation succeded (fallback, generally not shown)  */
+  /** The text when a generic operation succeeded (fallback, generally not shown)  */
   'panes.document-operation-results.operation-success': 'Utførte {{context}} på dokumentet',
-  /** The text when a delete operation succeded  */
+  /** The text when a delete operation succeeded  */
   'panes.document-operation-results.operation-success_delete': 'Dokumentet blei sletta',
   /** The text when a discard changes operation succeeded  */
   'panes.document-operation-results.operation-success_discardChanges':
     'Alle endringar sidan siste publisering har no blitt forkasta. Det forkasta utkastet kan framleis bli henta tilbake frå historikken',
   /** The text when a publish operation succeeded  */
   'panes.document-operation-results.operation-success_publish': 'Dokumentet blei publisert',
+  /** The text when a restore operation succeeded  */
+  'panes.document-operation-results.operation-success_restore':
+    '<Strong>{{title}}</Strong> ble gjenoppretta',
```

Comment:
Should be 

```
'<Strong>{{title}}</Strong> blei gjenoppretta'
```

or "vart" instead of "blei", but "blei" seems to be the consistent choice.